package.json

```diff
@@ -37,20 +37,27 @@
     "@types/jest": "^27.4.0",
     "@typescript-eslint/eslint-plugin": "^5.10.0",
     "@typescript-eslint/parser": "^5.10.0",
+    "axios": "^0.26.0",
+    "cookie-parser": "^1.4.6",
     "eslint": "^8.7.0",
     "eslint-config-prettier": "^8.3.0",
+    "http-status-codes": "^2.2.0",
     "husky": "^7.0.4",
     "jest": "^27.4.7",
     "lint-staged": "^12.2.2",
     "prettier": "2.5.1",
     "rollup": "^2.64.0",
     "ts-jest": "^27.1.3",
     "ts-node": "^10.4.0",
+    "tsconfig-paths": "^3.12.0",
     "tslib": "^2.3.1",
     "typescript": "^4.5.4"
   },
   "dependencies": {
     "@microsoft/api-extractor": "^7.19.4",
+    "@types/cookie-parser": "^1.4.2",
+    "form-data": "^4.0.0",
```

Comment:
@JFU-GIT Are these dependencies needed? I wasn't seeing `form-data` being used in the SDK, but might be missing it :-)

Reply:
TY for removing those!